src/win.c

```diff
@@ -1085,6 +1085,11 @@ static void win_determine_rounded_corners(session_t *ps, struct managed_win *w)
 	} else {
 		w->corner_radius = ps->o.corner_radius;
 		log_debug("Rounding corners for window %#010x", w->base.id);
+		// HACK: we reset this so we can query the color once
+		// we query the color in glx_round_corners_dst0 using glReadPixels
+		// w->border_col = { -1., -1, -1, -1 };
```

Comment:
Just for consistency, the function is now called `glx_round_corners_dst()` and the first value for `border_col` is written with a trailing dot.

Reply:
Ah, thanks for pointing it out!